stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelController.kt

```diff
@@ -65,7 +65,7 @@ public interface ChannelController {
     public fun getMessage(messageId: String): Message?
     // This one needs to be public for flows such as running a message action
     // TODO: this is for handling actions, think we should expose this in a different way
-    public fun upsertMessage(message: Message)
+    public suspend fun upsertMessage(message: Message)
```

Comment:
We should really address the comment above this method at this point. I don't think we have any suspending public API currently, and this change would introduce some. This would be uncallable from Java, for example.

Reply:
The upsert was there to handle actions. I think Sam added support for that a while ago. As far as I know we can make it internal.

Reply:
Actually, we don't need to expose this interface. I'd like to hide it (but it's not a reason of this PR). This interface do nothing without `ChatDomain`

Reply:
But that comment makes me not so sure even though I don't see the usage of this in our codebase.

Reply:
cc @samiuelson, do you know if we can make this internal? 😄

Reply:
IMO, it should be internal. No need for actions/command use case to have it public

Reply:
Awesome, let's do that then 👍